Samples/MyTestMacOSApp/MyTestMacOSApp/ADTestAppCache.m

```diff
@@ -141,7 +145,7 @@ - (OSStatus)readFromKeychain
         CFDataRef data = NULL;
         OSStatus status = SecItemCopyMatching((CFDictionaryRef)readQuery, (CFTypeRef *)&data);
         
-        _data = (__bridge NSData*)data;
+        _data = (__bridge_transfer NSData*)data;
```

Comment:
Here's another example where they're using a _bridge_retained for the query dictionary: http://stackoverflow.com/questions/7827730/iphone-fetch-data-dictionary-from-keychain